apps/design/frontend/src/ballot_order_info_screen.tsx

```diff
@@ -52,7 +52,7 @@ function BallotOrderInfoForm({
   }
 
   return (
-    <StyledForm>
+    <StyledForm onKeyUp={(e) => e.key === 'Enter' && onSaveButtonPress()}>
```

Comment:
This can be a later iteration, but usually better for a11y to:

1. use a `<button type="submit">` variant of the button element in the form and
2. override the `onSubmit` prop of the `<form>` element here and perform the action while also doing an `event.preventDefault()` to stop the browser from performing a default submit.

Enables default integration with things like mobile devices and some a11y devices.

Reply:
Thanks @kofi-q. I changed it to do as you suggested, including using `<button type="reset">` for the Cancel button.